spec/lib/miq_automation_engine/miq_ae_service_rbac_spec.rb

```diff
@@ -0,0 +1,213 @@
+include AutomationSpecHelper
+module MiqAeServiceModelSpec
+  include MiqAeEngine
+  describe MiqAeMethodService::MiqAeServiceVmOrTemplate do
+    before do
+      vm11
+      vm21
+      user1
+      user2
+    end
+
+    after do
+      Thread.current.thread_variable_set(:miq_rbac, false)
```

Comment:
can you use actual methods rather than setting thread current variables?

do note, that thread_variables are basically global variables. they happen to work better in a multi threaded context, but they are just as dangerous